keys/keys.go

```diff
@@ -88,6 +92,24 @@ func ImportPrivateKey(privKeyHex string, curve types.CurveType) (*KeyPair, error
 			PublicKey:  pubKey,
 			PrivateKey: rawPrivKey.Seed(),
 		}
+	case types.Secp256r1:
+		crv := elliptic.P256()
+		x, y := crv.ScalarBaseMult(privKey)
+		rawPubKey := ecdsa.PublicKey{X: x, Y: y, Curve: crv}
+		rawPrivKey := ecdsa.PrivateKey{
+			PublicKey: rawPubKey,
+			D:         new(big.Int).SetBytes(privKey),
+		}
+
+		pubKey := &types.PublicKey{
+			Bytes:     elliptic.Marshal(crv, rawPubKey.X, rawPubKey.Y),
+			CurveType: curve,
+		}
+
+		keyPair = &KeyPair{
+			PublicKey:  pubKey,
+			PrivateKey: rawPrivKey.D.Bytes(),
```

Comment:
Just to double check, is this 32 bytes? That'll be good so that we have uniformity with secp256k1, in which the keypair.PrivateKey is 32 bytes.

Reply:
Just checked myself, it is 32 bytes. https://play.golang.org/p/n0rt31c3lTs, so we are good here.

Reply:
How about big endian vs little endian in secp256k1 vs secp256r1?

Reply:
Keys and signatures of both curves are encoded in big-endian form